components/LocationComparisonSection/LocationTagsListItem.tsx

```diff
@@ -0,0 +1,29 @@
+import React, { FunctionComponent } from 'react';
+
+interface LocationTagsListItemProps {
+  label: string;
+  active?: boolean;
+  removeTag: (label: string) => void;
+}
+
+const LocationTagsListItem: FunctionComponent<LocationTagsListItemProps> = ({ label, removeTag }) => {
+  const remove = (): void => {
```

Comment:
`onRemove`

Reply:
Updated to match event naming conventions